MentoringHomeWork/HW9/AirportSimulation/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿namespace AirportSimulation;
+
+public class Program
+{
+    public static void Main()
+    {
+        string? passportNumber;
+        while (true)
+        {
+            Console.Write("Enter Passport Number: ");
+            passportNumber = Console.ReadLine();
+
+            if (!string.IsNullOrWhiteSpace(passportNumber))
+            {
+                break;
+            }
+            else
```

Comment:
'Else' statement is not required here - also you have suggestions about it from IntelliSense. In real life, you will see the next code.

```suggestion
            if (!string.IsNullOrWhiteSpace(passportNumber))
            {
                break;
            }

            Console.WriteLine("Passport number cannot be empty. Please enter a valid passport number.");
```

Reply:
resolved